README.md

```diff
@@ -375,6 +375,23 @@ config: {
 ```
 the latter will be applied and the map view will be moved the defined coordinates.
 
+### Github pages
+We currently host the demo-app on Github pages. We have provided a way to test github pages before pushing the branch to 
+the actual repo.
+In order to test github pages with your changes, you need to satisfy the following requirements first:
+- Make sure you have your own github pages (username.github.io) repo, [click here](https://pages.github.com/)
+- In your local copy of kepler.gl, add your github pages repo to the list of git remotes by doing:
```

Comment:
`copy` -> `fork` for clarity? (don't want to do this in a clone of kepler, correct? or does it matter?)

Reply:
Move this part to DEVELOPERS.md

Reply:
@ericsoco it doesn't really matter since developers will only change their local git configuration

Reply:
> Move this part to DEVELOPERS.md

done